.gitignore

```diff
@@ -24,3 +24,9 @@ en_OECD.*
 
 # OS files
 .DS_Store
+
```

Comment:
Revert these changes to `.gitignore` please.

Reply:
This has been fixed.  Why won't it go away?
Is there more than 1 .gitignore?